src/user-icon/user-icon.scss

```diff
@@ -81,8 +81,8 @@
       @include themed {
         fill: t(iui-icons-color);
       }
-      width: calc(100% / 1.5);
-      height: calc(100% / 1.5);
+      width: calc(100%* 0.6666667);
+      height: calc(100%* 0.6666667);
```

Comment:
This really gives the same value? 😮

Reply:
yep 1/1.5 = 0.6666667 - I messed up spacing I will fix that.

Reply:
Well, I'm more concerned about the precision

Reply:
I have updated to use 0.66666666666666667 which produces the same results as 1/1.5 in test values I tried from 1 - 9600.